src/docs/product/accounts/early-adopter-features/issue-archiving.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Issue Archiving"
+sidebar_order: 1
+description: "Learn how issue status works and and how to triage issues."
+---
+
+Use the status tags attached to issues on the [**Issues** page](https://sentry.io/issues/) in Sentry to help you triage and prioritize problems with your application that are important to you. Keep in mind that an issue can only have **one status at a time.**
+
+Here's a list of all statuses, how they're assigned to an issue, and their custom search term:
+
+| Status         | Condition                                                                                                                     | Custom Search Term |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| **New**        | An issue that was created in the last 7 days.                                                                                 | is:new             |
+| **Ongoing**    | An issue that was created more than 7 days ago or has manually been marked as reviewed.                                       | is:ongoing         |
+| **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). | is:escalating      |
```

Comment:
@lizokm - where should the Escalating Issues Algorithm link to?

Reply:
Ah! Great find.